challenges.yml

```diff
@@ -193,3 +193,245 @@ stimulus_values:
         d: |
           `this.values.colorSelected`;
       correct: c
+value_change_callback:
+  question: |
+    Suppose we have a Stimulus controller like this:
+
+    ```js
+    export default class extends Controller {
+        static targets = ['selectItem'];
+
+        static values = {
+            itemId: Number
+        }
+
+        connect() {
+            console.log("Hey I'm connected!");
+        }
+    }
+    ```
+    How would you add a *change callback* method to the `itemId` value?
+  explanation: |
+    The "Value Change Callback" feature of the Stimulus values API implements a naming convention. The method's name
+    must be equal to the name of the *value* property plus the word `ValueChanged`.
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          ```js
+          itemIdChanged() {
+              ...
+          }
+          ```
+        b: |
+          ```js
+          itemIdValueChanged() {
+              ...
+          }
+          ```
+        c: |
+          ```js
+          onChangeItemIdValue() {
+              ...
+          }
+          ```
+        d: |
+          ```js
+          itemIdValue() {
+              ...
+          }
+          ```
+      correct: b
+search_preview:
+  question: |
+    How can you pass data to a Stimulus controller from a Twig template using the Values API?
```

Comment:
I would go just a TINY bit further. Add a simple code example (like you did above) showing an `id` value. Then ask how they could pass in this value. Below, one of the wrong answers could be `idValue: ...`

Oooh, and an even better example would be a value that is two words - like the `itemId` you had earlier. That's because (and we could mention this in the explanation), you can pass `itemId:` to `stimulus_controller`, but ultimately, it renders as `data-my-controller-name-item-id-value="10">` where the `itemId` is normalized to `item-id`. The function handles that for you.